### DISCUSSION TITLE
When are public shares discarded?

### DISCUSSION THREAD

**Discussion by martin-vitek** [photoview/photoview#304] [Q&A and Troubleshooting] [answered]
Under which conditions are links to public shares deleted?

**Comment by viktorstrate** [comment, marked as answer]
When you delete them from the list.

<img width="485" alt="Screenshot 2021-04-08 at 19 29 23" src="https://user-images.githubusercontent.com/4233458/114070839-bb548580-98a0-11eb-8fd5-41e30536955a.png">

**Reply by martin-vitek** [reply]
Yes, I know. I meant automatically like I guess when an album is renamed? I had a situation when I added a new album on the filesystem and after rescanning all photos, share to another album (unmodified) was gone.

**Reply by viktorstrate** [reply]
It should only be deleted if one of the parent directories or the album directory itself is renamed.
Basically if the filepath to the album changes, Photoview will see it as a new album.

**Reply by martin-vitek** [reply]
Ok, thank you. It is possible, that the share went away when I was doing rescanning, selinux didn't allow Photoview access to the photos and it all went to hell.
